libs/ledger/include/ledger/chain/block.hpp

```diff
@@ -61,6 +61,7 @@ class Block
     Slices   slices;             ///< The slice lists
     DAGEpoch dag_epoch;          ///< DAG epoch containing information on new dag_nodes
     uint64_t timestamp{0u};      ///< The number of seconds elapsed since the Unix epoch
+    uint64_t random_beacon{0u};  ///< Entropy that determines miner priority for the next block
```

Comment:
Maybe we could rename this to `block_entropy` or something?

Reply:
ok, the naming of random beacon will be gone